editor/src/components/editor/server.ts

```diff
@@ -377,7 +384,29 @@ export function startPollingLoginState(
   setInterval(async () => {
     const loginState = await getLoginState('no-cache')
     if (previousLoginState.type !== loginState.type) {
+      if (isLoggedIn(loginState)) {
+        // Fetch the user configuration
+        void getUserConfiguration(loginState).then((userConfig) =>
```

Comment:
We probably should `await` these, otherwise if they do a go slow or block, this could pile up multiple calls all on top of each other. I believe that they wont block the main thread if done that way either.

Reply:
I'm not sure if `setInterval` is smart enough to wait until Xms _after_ the last one fired or not....

Reply:
These only fire if the login state has changed. My thinking was to not `await` them so that we don't have to block updating the login state, user config, or github authentication state - they can each be updated as soon as we have the data for them.

Reply:
That's reasonable.